llvm/test/CodeGen/RISCV/rvv/vp-reduce.ll

```diff
@@ -0,0 +1,205 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
This is an overly generic test name. How are the other vp.reduce tests structured?

Reply:
I'd consolidates the tests with other floating point vp.reduce tests.